explain/setup/index.mdx

```diff
@@ -29,6 +38,17 @@ Google Cloud SQL | Yes               | No                |
 Heroku Postgres  | Yes               | No                |
 Self-managed VM  | Yes               | Yes (Recommended) |
```

Comment:
My gut feeling is it might be confusing to have links from both `Yes`es on platforms that support both (it can only steer users away from the recommended option) and it might also be confusing to only have a link to only the recommended option ("If the other option is supported, how do I get to those instructions?"). Given that the actual links are right below, I think I'd rather leave them out of this table. Thoughts?